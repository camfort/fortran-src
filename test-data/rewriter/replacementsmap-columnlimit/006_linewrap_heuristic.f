      subroutine foo
      integer word, wrap_after_comma, wrap_after_right_paren
      logical accessor, TEST, LOGICAL2VAR
      if (accessor(word, wrap_after_comma)) call bar
      if (accessor(int(word), wrap_after_comma)) call bar
      if (accessor(int(int2(word)), wrap_after_comma)) call bar
      if (accessor(int(int2(word)))) call wrap_after_right_paren
      if (accessor(word)) call foo(bar)!don't wrap inline comment
      accessor(1) = TEST!inline comment not to be wrapped

      if(1) then
            LOGICAL2VAR = l2!other comment that shouldn't be wrapped
            !comments after 0! char replacements don't wrap!
            L!comments after replacements >0 chars don't wrap
      endif
      end
