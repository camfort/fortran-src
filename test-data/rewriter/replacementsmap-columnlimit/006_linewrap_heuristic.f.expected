      subroutine foo
      integer word, wrap_after_comma, wrap_after_right_paren
      logical accessor, TEST, LOGICAL2VAR
      if (accessor(xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx,
     + wrap_after_comma)) call bar
      if (accessor(int(xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx),
     + wrap_after_comma)) call bar
      if (accessor(int(int2(xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx)),
     + wrap_after_comma)) call bar
      if (accessor(int(int2(xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx))))
     + call wrap_after_right_paren
      if (accessor(xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx))
     + call foo(bar)!don't wrap inline comment
      call testcommons_logarray_settori_findex(logical_1_to_4(test), 1)!inline comment not to be wrapped

      if(1) then
            call testcommons_set_logical2var(logical_2_to_4(l2))!other comment that shouldn't be wrapped
            call testcommons_set_logical2var(logical_2_to_4(l2))!comments after 0! char replacements don't wrap!
            call testcommons_set_logical2var(logical_2_to_4(l2))!comments after replacements >0 chars don't wrap
      endif
      end
